Cargo.toml

```diff
@@ -82,6 +82,10 @@ name = "rhai-run"
 name = "rhai-dbg"
 required-features = ["debugging"]
 
+[[example]]
```

Comment:
I remember that, in Rust, if you put one `[[example]]`, you need to list all examples and it'd throw away the auto-generated list.

I'm not very sure though, but this can easily be done by putting a feature gate on the example file.  Quite a few examples which depend on features are done this way.

This would also make all the tests pass regardless of features.

Reply:
Does not seem to be the case, at least in 1.61, the examples seem to work fine.

Reply:
Hhmmm.... I remember something with breaking CI with different features if I use `required-features`.  Let me check that out...

Reply:
The CI seems to pass, there are only some issues where I use feature-gated items.

Reply:
I checked. It is OK!  So go ahead!

Reply:
BTW, I remember now.  There is `required-features` which works, but no `cannot-have-features` so for most of my examples I'd need to put the feature gate in the file itself.